### PR TITLE
WEEEOOOO WEEEEOOO WEEEOOO

### PR DESCRIPTION
**Pull request by dannno**

### CODE REVIEW

**Comment by Iamgoofball** [comment]
best feature from NoX, merge ASAP

:+1: x :100:

**Comment by An0n3** [comment]
WOOP WOOP

ITS DA SOUND OF DA POLICE

**Comment by WJohn** [comment]
Oh god yes it's getting added for real.

**Comment by MrPerson** [comment]
I hope  6 months from now some spriter is like "wow the siren hat looks like shit" and gives us something better.

Until then of course, it's hard not to enjoy the siren hat.

**Comment by Razharas** [comment]
>No sound

0/5

**Comment by dannno** [comment]
get me a WEEOOOO sound

**Comment by Dorsisdwarf** [comment]
http://forum.emergency-planet.com/tutorials/article/30-creating-your-own-sirens/

**Comment by Nienhaus** [comment]
:+1:

**Comment by SconesC** [comment]
WEOOOO WEOOOO

**Comment by dannno** [comment]
Clothing code is fucking awful.
I'll clean it up at some point maybe.

**Comment by Razharas** [comment]
>32 seconds of repeating sound

Cut a single weeoo out and play it in loop by code

**Comment by dannno** [comment]
how

**Comment by Razharas** [comment]
while(active)
     play_sound(your_sound)
     sleep(your_sound_length_in_seconds)

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -53,23 +56,39 @@
...
+				usr.update_inv_head(0)
+			else
+				up = !up
+				flags &= ~(visor_flags)
+				flags_inv &= ~(visor_flags_inv)
+				icon_state = "[initial(icon_state)]up"
+				usr << "[alt_toggle_message]"
+				usr.update_inv_head(0)
+				playsound(src.loc, "[activation_sound]", 100, 0, 4)
+
```
Comment (Sometinyprick) [inline code/modules/clothing/head/helmet.dm]: wouldn't this  play a siren noise for the riot helmet too?
or I'm I being dumb

Comment (dannno) [inline code/modules/clothing/head/helmet.dm]: activation_sound is null for the riot helmet, so no.

**Comment by Czarcastic** [comment]
It would look funnier with just a single light like an ape escape helmet tbh

**Comment by WJohn** [comment]
Because we have fashion police, we clearly need that as the alternative, since there's already an alternative for EVERY SINGLE PIECE OF CLOTHING IN SEC.

File: code/modules/clothing/head/helmet.dm (hunk 1 above)
Comment (Sometinyprick) [inline code/modules/clothing/head/helmet.dm]: ok, was me being dumb

**Comment by dannno** [comment]
@Razharas it's 1.4 seconds tho

**Comment by GunHog** [comment]
Do you think you could have these allow mounting of a camera?  Also, these are massive rotating police lights, make sure they give luminosity when active!

**Comment by Razharas** [comment]
Sleep(14) it is then

**Comment by Kraseo** [comment]
@WJohn because we still need more xeno sprites don't we

**Comment by WJohn** [comment]
Yes.

**Comment by Jordie0608** [comment]
Sounds doesn't loop all that cleanly but better than no sound at all :+1: 
Also changelog cos why not

**Comment by dannno** [comment]
I thought it looped pretty cleanly, it was the best I could manage

**Comment by RemieRichards** [comment]
Somebody sprite one of these with a single siren on like in ape escape (as was suggested further up)
since I just did: https://github.com/tgstation/-tg-station/pull/8554

**Comment by dannno** [comment]
see I wasn't motivated to do it but now I absolutely will.

**Comment by Steelpoint** [comment]
Bit derpy looking and its not consistent with our current helmet design.

But its pretty nifty anyway, so :+1:

**Comment by GunHog** [comment]
Make sure these hats actually emit light. Two HUGE, POLICE CAR sized lights having zero luminosity would be...disappointing.

**Comment by dannno** [comment]
eh, the thing with that is that we don't have colored lighting so it would be a bit weiiirdd...

**Comment by WJohn** [comment]
Doesn't need colored light.

**Comment by dannno** [comment]
I tried adding luminosity and it broke

**Comment by Miauw62** [comment]
that sprite is 10/10

**Comment by dannno** [comment]
conflicts fixed

**Pull request merged**